### DISCUSSION TITLE
我想让WEB上默认使用我私有部署的API，请问应该在哪里配置？

### DISCUSSION THREAD

**Discussion by erquren** [ChatGPTNextWeb/NextChat#4364] [Q&A]
我没有查询到配置的参数，现在是通过复制连接里加上url来进行配置，但是这样不美观，请问有什么别的办法吗？

**Comment by erquren** [comment]
我当前采用的是docker部署
docker run -d -p 3000:80 yidadaa/chatgpt-next-web

例如我的API地址为test.com/api/ai/ ,请问应该在哪里配置？

**Comment by goodls123** [comment]
我是在clien目录自己写了一个

**Reply by erquren** [reply]
这倒是一个办法
